src/site/content/en/learn/css/spacing/index.md

```diff
@@ -0,0 +1,380 @@
+each side of your element will have a computed margin of `50px`.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'YzNQeyb'
+} %}
+
+You can also use a value of `auto` for margin.
+For block level elements with a restricted size,
+an `auto` margin will take up available space in the direction that it is applied to.
+A good example is this one,
+from the [flexbox module](/learn/css/flexbox), where the items push away from each other.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'poRELbR'
+} %}
+
+Another good example of `auto` margin is a horizontally centered wrapper which has a max width.
+This sort of wrapper is often used to create a consistent center column on a website.
+
+```css
+.wrapper {
+	max-width: 400px;
+	margin: 0 auto;
+}
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'XWpgZKq'
+} %}
+
+Here, margin is removed from the top and bottom (block) sides,
+and `auto` shares the space between the left and right (inline) sides.
+
+{% Aside %}
+In the last lesson on [logical properties](/learn/css/logical-properties),
+you learned that instead of specifying `margin-top`, `margin-right`, `margin-bottom` and `margin-left`,
+you can use `margin-block-start`, `margin-inline-end`, `margin-block-end` and `margin-inline-start`.
+{% endAside %}
+
+### Negative margin
+
+Negative values can also be used for margin.
+Instead of adding space between adjacent sibling elements,
+it will **reduce space** between them.
+This can result in overlapping elements,
+if you declare a negative value that's more than the available space.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'YzNQeLP'
+} %}
+
+### Margin collapse
+
+Margin collapse is a tricky concept,
+but it's something you'll run into very commonly when building interfaces.
+Say you have two elements: a heading and a paragraph that both have vertical margin on them:
+
+```html
+<article>
+  <h1>My heading with teal margin</h1>
+  <p>A paragraph of text that has blue margin on it, following the heading with margin.</p>
+</article>
+```
+
+```css
+h1 {
+	margin-bottom: 2rem;
+}
+
+p {
+	margin-top: 3rem;
+}
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'OJWgvMx'
+} %}
+
+At first glance,
+you would be forgiven for thinking that the paragraph will be spaced `5em` from the heading,
+because `2rem` and `3rem` combined calculate to `5rem`.
+Because **vertical margin collapses**, though, the space is actually `3rem`.
+
+Margin collapse works by selecting the largest value of two adjoining elements
+with vertical margin set on the adjoining sides.
+The bottom of the `h1` meets the top of the `p`,
+so the largest value of the `h1`'s bottom margin and the `p`'s top margin is selected.
+If the `h1` were to have `3.5rem` of bottom margin,
+the space between them both would then be `3.5rem` because that is larger than `3rem`.
+Only block margins collapse, not inline (horizontal) margins.
+
+Margin collapse is rooted back to when the web was mostly just documents.
```

Comment:
```suggestion
This concept is rooted back to when the web was mostly just documents.
```

These 3 paragraphs all start with "margin collapse", so im thinking on how to change it. I also this this could be an aside element to break up the text blocks

Reply:
good call, I've made it an aside in my recent commit.